yktracec/src/jitmodbuilder.cc

```diff
@@ -1273,6 +1301,12 @@ class JITModBuilder {
               // setjmp/longjmp, so for now simply abort this trace.
               // See: https://github.com/ykjit/yk/issues/610
               return nullptr;
+            } else if (IsSWTrace && S == "yk_trace_basicblock") {
+              // YKFIXME: We don't want jitmodbuilder to be aware of any
```

Comment:
This is a very opaque comment. Can we clarify?

Reply:
Updated 👉 a79c916cc989e4bf8d3dda072e71fd30da3839f3
Is it clearer?